src/NLog/Layouts/JsonLayout.cs

```diff
@@ -53,6 +53,7 @@ public class JsonLayout : Layout
         public JsonLayout()
         {
             this.Attributes = new List<JsonAttribute>();
+            this.RenderEmptyLiteral = true;
```

Comment:
That's why i set it to true by default. The behaviour was to render empty literals by default, and I wanted an option of removing them, so one has to opt out by setting it to false.